linux_os/guide/system/network/network-firewalld/ruleset_modifications/configure_firewalld_ports/rule.yml

```diff
@@ -63,3 +61,10 @@ ocil: |-
     Inspect the list of enabled firewall ports and verify they are configured
     correctly by running the following command:
     <pre>$ sudo firewall-cmd --list-all</pre>
+
+fixtext: |-
+    Configure {{{ full_name }}} to allow approved services to have access to the system.
+    To open a port for a service, configure firewalld using the following command:
+    $ sudo firewall-cmd --permanent --add-port=port_number/tcp
```

Comment:
Is the standard to show commands with `sudo`? I saw cases with and without `sudo`. We should agree in a standard and use everywhere. What do you think?

Reply:
I think that we should use sudo everywhere in OCILs and fixtexts. The RHEL 8 table that Becker posted https://complianceascode.github.io/content/tables/table-rhel8-stig-manual.html uses sudo almost everywhere,